core/roboconf-dm-rest-services/src/main/java/net/roboconf/dm/rest/services/internal/resources/impl/ManagementResource.java

```diff
@@ -279,4 +292,102 @@ public Response shutdownApplication( String applicationName ) {
 
 		return result;
 	}
+
+
+	@Override
+	public Response setImage(final String name, final String qualifier, final InputStream image,
+							 final FormDataContentDisposition fileDetail) {
+
+		// Do set the image, and wrap exceptions in a HTTP response.
+		Response response;
+		try {
+			doSetImage(name, qualifier, image, fileDetail);
+			response = Response.noContent().build();
+		} catch (final NoSuchElementException | IllegalArgumentException e) {
```

Comment:
I might potentially admit this is my own interpretation of the *Holy HTTP specification*. :)